src/lib/snark_work_lib/selector.ml

```diff
@@ -0,0 +1,82 @@
+open Core_kernel
```

Comment:
Could we add a doc-comment for the file, what's selector types are made for?

Reply:
Will do so :)